Services/Dashboard/Administration/classes/class.ilDashboardSortationTableGUI.php

```diff
@@ -0,0 +1,105 @@
+        $this->lng->loadLanguageModule('mme');
+        $this->initColumns();
+        $this->setFormAction($this->ctrl->getFormAction($this->parent_obj));
+        $this->addCommandButton('saveSettings', $this->lng->txt('save'));
+        $this->setRowTemplate(
+            "tpl.dashboard_sortation_row.html",
+            "Services/Dashboard"
+        );
+        $this->setEnableNumInfo(false);
+        $this->initData();
+    }
+
+    public function initColumns(): void
+    {
+        $this->addColumn($this->lng->txt('topitem_position'), '', '30px');
+        $this->addColumn($this->lng->txt('topitem_title'));
+        $this->addColumn($this->lng->txt('topitem_active'));
+    }
+
+    public function initData(): void
+    {
+        $data[] = [
+            'position' => $this->uiRenderer->render($this->uiFactory->divider()->horizontal()->withLabel($this->lng->txt("dash_main_panel"))),
+            'title' => '',
+            'active_checkbox' => '',
+        ];
+
+        $position = 0;
+        foreach ($this->viewSettings->getViewPositions() as $presentation_view) {
+            $presentation_cb = new ilCheckboxInputGUI('', 'main_panel[enable][' . $presentation_view . ']');
+            $presentation_cb->setChecked($this->viewSettings->isViewEnabled($presentation_view));
+            $presentation_cb->setValue('1');
+            $presentation_cb->setDisabled($presentation_view === ilPDSelectedItemsBlockConstants::VIEW_RECOMMENDED_CONTENT);
+
+            $position_input = new ilTextInputGUI('', 'main_panel[position][' . $presentation_view . ']');
+            $position_input->setSize(3);
+            $position_input->setValue((string) (++$position * 10));
+
+            $data[] = [
+                'position' => $position_input->render(),
+                'title' => $this->lng->txt("dash_enable_" . $this->viewSettings->getViewName($presentation_view)),
+                'active_checkbox' => $presentation_cb->render(),
+            ];
+        }
+
+        $data[] = [
+                'position' => $this->uiRenderer->render($this->uiFactory->divider()->horizontal()->withLabel($this->lng->txt("dash_side_panel"))),
+                'title' => '',
+                'active_checkbox' => '',
+            ];
+
+        $sp_fields = [];
+        $position = 0;
+        foreach ($this->side_panel_settings->getPositions() as $mod) {
+            $side_panel_module_cb = new ilCheckboxInputGUI('', 'side_panel[enable][' . $mod . ']');
+            $side_panel_module_cb->setChecked($this->side_panel_settings->isEnabled($mod));
+
+            $position_input = new ilTextInputGUI('', 'side_panel[position][' . $mod . ']');
+            $position_input->setValue((string) (++$position * 10));
+
+            $data[] = [
+                'position' => $position_input->render(),
+                'title' => $this->lng->txt("dash_enable_" . $mod),
+                'active_checkbox' => $side_panel_module_cb->render(),
+                ];
```

Comment:
```suggestion
            ];
```

You **CAN** prevent such indentation errors by removing the last `,` in your formatted array.
This helps most IDEs to detect your end of array and therefore automatically fix the indentation.